Library/Homebrew/utils/formatter.rb

```diff
@@ -7,6 +7,9 @@
 #
 # @api private
 module Formatter
+  COMMAND_DESC_WIDTH = 80
+  OPTION_DESC_WIDTH = 45
```

Comment:
These are only used in calls to `Formatter`, so seemed to make the most sense here